opensanca/opensanca_jobs
# Company registration and login

This is the very beginning part of the company dashboard.
This commit adds the company registration and login and requires login to
register a new job.

Closes #64 

![image](https://user-images.githubusercontent.com/771411/36496502-3b5e0f80-1717-11e8-8c1d-1d66d5b6a0a1.png)

![image](https://user-images.githubusercontent.com/771411/36496509-4448921e-1717-11e8-80eb-92d490fbf12d.png)

# How to test

- Visit the root page
- Click on `Anunciar Vaga` button.
It should redirect to the login page

- Click on `Registrar` link, and set a new account.
It should create your account and login for you.

- Crate a new Job.
It should create a new job, redirect to the job page. The page should contain the company information (from the Registration page)

- Click on Logout
It should log out with success. 

You should visit the job show page without login.
